payments/models.py

```diff
@@ -37,8 +37,8 @@ def total_tickets_sold(self):
 
     @property
     def is_open(self):
-        return self.rate_set.filter(
-            private=False, sold_out=False, sales_open=True)
+        return len(self.rate_set.filter(
```

Comment:
I think the recommended way to check the size of a queryset is to use the [`count()` method](http://stackoverflow.com/questions/14327036/count-vs-len-on-a-django-queryset).